src/applications/vaos/services/appointment/index.js

```diff
@@ -214,6 +214,17 @@ export function isVAPhoneAppointment(appointment) {
   return appointment?.vaos.isPhoneAppointment;
 }
 
+/**
+ * Returns whether or not the appointment is COVID vaccine appointment
+ *
+ * @export
+ * @param {Object} appointment A FHIR appointment resource
+ * @returns {Boolean} Whether or not the appointment is for COVID vaccine
+ */
+export function isCOVIDVaccineAppointment(appointment) {
+  return appointment?.vaos.isCOVIDVaccine;
```

Comment:
I would rather not have a helper for this, since it's a just accessing a particular property on the Appointment resource. If you feel like the name of the property wasn't clear to you, we could definitely update it.

Reply:
Ok, makes sense. I just was trying to optimize but that makes sense in regards to the property. 👍